src/code/templates/genome-challenge.js

```diff
@@ -84,6 +84,8 @@ export default class GenomeChallengeTemplate extends Component {
   static authoredDrakesToDrakeArray = function(authoredChallenge, trial) {
     if (authoredChallenge.trialGenerator) {
       return generateTrialDrakes(authoredChallenge.trialGenerator, trial);
+    } else if (authoredChallenge.initialDrake.constructor === Array) {
```

Comment:
Array.isArray() is more robust and is supported by all browsers we care about (cf. [Determining with absolute accuracy whether or not a JavaScript object is an array](http://web.mit.edu/jwalden/www/isArray.html)).